modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/MediaSessionManager.kt

```diff
@@ -124,6 +145,14 @@ class MediaSessionManager(
         )
 
         connect()
+
+        @OptIn(DelicateCoroutinesApi::class)
+        GlobalScope.launch {
```

Comment:
My initial intention was to avoid using GlobalScope here thanks to the changes in https://github.com/Automattic/pocket-casts-android/pull/1619, but when I decided to have this PR target the release branch I dropped that plan because I do not think we want to add the changes in https://github.com/Automattic/pocket-casts-android/pull/1619 to the release branch. We can just update this usage of Global Scope to use the applicationScope from https://github.com/Automattic/pocket-casts-android/pull/1619 after we merge these changes to `main`.